quiche/src/recovery/gcongestion/bbr2.rs

```diff
@@ -179,14 +181,18 @@ const PARAMS: Params = Params {
 
     max_startup_queue_rounds: 0,
 
+    // https://github.com/google/quiche/blob/98c9cdb4cd17ea043243037bfdee3cdf024cab54/quiche/common/quiche_protocol_flags_list.h#L151-L153
     startup_full_loss_count: 8,
 
     drain_cwnd_gain: 2.0,
 
-    drain_pacing_gain: 1.0 / 2.885,
+    // The startup_pacing_gain is 2.773
+    drain_pacing_gain: 1.0 / 2.773,
```

Comment:
The drain seem too conservative currently

Reply:
Where does this tweak from?

I see:
`quiche/quic/core/congestion_control/bbr2_misc.h:  float drain_pacing_gain = 1.0 / 2.885;`